tests/src/Interop/COM/NativeClients/Primitives/ArrayTests.cpp

```diff
@@ -3,6 +3,8 @@
 // See the LICENSE file in the project root for more information.
 
 #include "ClientTests.h"
+#include <stdio.h>
```

Comment:
Could you instead make that `#include <cstdio>`?

Reply:
I can, but this file says `::printf` not `std::printf`, and the standard doesn't require that `<cstdio>` provides that (though in practice I know of no implementation that does not provide it). Do you want those fixed up too?

Reply:
oh, that is right - boo for me not following the standard. You can leave this as is. Thanks.